src/main/java/uk/ac/ebi/spot/gwas/curation/rest/PublicationsController.java

```diff
@@ -93,4 +124,11 @@ public Resource<PublicationDto> getPublication(@PathVariable String id) {
         }
     }
 
+    @ResponseStatus(HttpStatus.OK)
+    @PutMapping(value = "/{pmid}/link-submission")
+    @PreAuthorize("hasRole('self.GWAS_Curator')")
+    public void linkSubmission(@PathVariable String pmid, @RequestParam String submissionId) {
+        publicationService.linkSubmission(pmid, submissionId);
+    };
```

Comment:
The trailing ";" will constitute an empty stament & code smell at the close of the metthind 
This should be removed